synced-files/.github/workflows/no-response.yml

```diff
@@ -0,0 +1,22 @@
+# NOTE: This is a common file that is overwritten by realm/ci-actions sync service
+# and should only be modified in that repository.
+
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
```

Comment:
Should we also use a fixed commit here?

Reply:
Unfortunately, that action is packaged weirdly so only tagged releases have the actual executable code built.